src/EnergyPlus/NodeInputManager.cc

```diff
@@ -176,16 +171,17 @@ void GetNodeNums(EnergyPlusData &state,
     }
 
     // Most calls to this routine use a fixed fluid stream number for all nodes, this is the default
-    FluidStreamNum = NodeFluidStream;
+    NodeInputManager::CompFluidStream FluidStreamNum = NodeFluidStream;
+    DataLoopNode::ConnectionType thisConnectionType;
     for (int Loop = 1; Loop <= NumNodes; ++Loop) {
 
         switch (nodeConnectionType) {
         case DataLoopNode::ConnectionType::Num:
         case DataLoopNode::ConnectionType::Invalid:
-            ConnectionType = ConnectionType::Invalid;
+            thisConnectionType = ConnectionType::Invalid;
             break;
         default:
-            ConnectionType = nodeConnectionType;
+            thisConnectionType = nodeConnectionType;
             break;
         }
```

Comment:
Doesn't the connection type default to invalid? And isn't thisConnectionType always equal to nodeConnectionType (except for Num which should never happen)?